stable/redis/values-production.yaml

```diff
@@ -0,0 +1,230 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis
+imageTag: 4.0.9-r0
+
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+imagePullPolicy: IfNotPresent
+
+## Image credentials
+# imageCredentials:
+  # registry: docker.io
+  # username: javsalgar
+  # password: password
+
+## Cluster settings
+cluster:
+  enabled: true
+  slaveCount: 3
+
+## Use existing secret (ignores previous password)
+# existingSecret:
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistence:
+  enabled: true
+
+  ## The path the volume will be mounted at, useful when using different
+  ## Redis images.
+  path: /bitnami
```

Comment:
You're persisting the configuration of Redis when using Bitnami Image. Please persist only the data directory.
This also applies to `values.yaml`